content/en/agent/guide/private-link.md

```diff
@@ -28,75 +24,168 @@ The overall process consists of configuring an internal endpoint in your VPC tha
+    **Note**: **The security group must accept inbound traffic on TCP port `443`**.
 
-| Datadog API Service Name                                  |
-| --------------------------------------------------------- |
-| `com.amazonaws.vpce.us-east-1.vpce-svc-064ea718f8d0ead77` |
+8. Hit **Create endpoint** at the bottom of the screen. If successful, the following is displayed:
 
-{{% /tab %}}
-{{% tab "Processes" %}}
+   {{< img src="agent/guide/private_link/vpc_endpoint_created.png" alt="VPC endpoint created" style="width:60%;" >}}
 
-| Datadog Process Monitoring Service Name                   |
-| --------------------------------------------------------- |
-| `com.amazonaws.vpce.us-east-1.vpce-svc-0ed1f789ac6b0bde1` |
+9. Click on the VPC endpoint ID to check its status.
+10. Wait for the status to move from _Pending_ to _Available_. This can take up to 10 minutes. Once it shows _Available_, the AWS PrivateLink is ready to be used.
 
-{{% /tab %}}
-{{% tab "Traces" %}}
+    {{< img src="agent/guide/private_link/vpc_status.png" alt="VPC status" style="width:60%;" >}}
 
-| Datadog Trace Service Name                                |
-| --------------------------------------------------------- |
-| `com.amazonaws.vpce.us-east-1.vpce-svc-0355bb1880dfa09c2` |
+11. If you are collecting logs data, ensure your Agent is configured to send logs over HTTPS. If the data is not already there, add the following to the [Agent `datadog.yaml` configuration file][2]:
 
-{{% /tab %}}
-{{% tab "Kubernetes" %}}
+    ```yaml
+    logs_config:
+        use_http: true
+    ```
+
+    If you are using the container Agent, set the following environment variable instead:
+
+    ```
+    DD_LOGS_CONFIG_USE_HTTP=true
+    ```
+
+    This configuration is required when sending logs to Datadog with AWS PrivateLink and the Datadog Agent, and is not required for the Lambda Extension. See [Agent log collection][3] for more details.
+12. [Restart your Agent][4] to send data to Datadog through AWS PrivateLink.
 
-| Datadog Kubernetes Explorer Service Name                  |
-| --------------------------------------------------------- |
-| `com.amazonaws.vpce.us-east-1.vpce-svc-0ad5fb9e71f85fe99` |
 
+
+[1]: /help/
+[2]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
+[3]: /agent/logs/?tab=tailexistingfiles#send-logs-over-https
+[4]: /agent/guide/agent-commands/#restart-the-agent
 {{% /tab %}}
-{{< /tabs >}}
 
-4. Click **Verify**. If this does not return _Service name found_, reach out to [Datadog support][2].
-5. Choose the VPC and subnets that should be peered with the Datadog VPC service endpoint.
-6. Make sure that for **Enable DNS name** the _Enable for this endpoint_ is checked:
-   {{< img src="agent/guide/private_link/enabled_dns_private.png" alt="Enable DNS private" style="width:60%;" >}}
-7. Choose the security group of your choice to control what can send traffic to this VPC endpoint.
+{{% tab "VPC peering" %}}
+
+### Amazon VPC peering
```

Comment:
Do we need this H3 if the tab says "VCP peering?"

```suggestion
### Amazon VPC peering
```

Reply:
Yeah, this one is necessary as there are 2 different areas you need to configure in Console: VPC Peering and Route 54, then you have to configure in Datadog. It's 3 separate areas of configuration, hence the headers.